bot/exts/moderation/modlog.py

```diff
@@ -522,14 +521,32 @@ async def log_cached_deleted_message(self, message: discord.Message) -> None:
                 f"**Message ID:** `{message.id}`\n"
                 f"**Sent at:** {format_dt(message.created_at)}\n"
                 f"[Jump to message]({message.jump_url})\n"
-                "\n"
             )
 
+        # If the message is a reply, add the reference to the response
+        if message.reference is not None and message.reference.resolved is not None:
```

Comment:
Even if the message reference could not b resolved, we can still have an indication that the deleted message was a reply.
So something like:
`In reply to: (Message could not be resolved)`